services/Cargo.toml

```diff
@@ -37,6 +37,7 @@ gdal = "0.15.0"
 gdal-sys = "0.9"
 geo = "0.26"
 geoengine-datatypes = { path = "../datatypes" }
+geoenginemacros = { path = "../macros" }
```

Comment:
shouldn't that be "geoengine_macros" if we cant use "geoengine-macros"?

Reply:
[4c5dee7](https://github.com/geo-engine/geoengine/pull/859/commits/4c5dee7e753d5e3246257042fa28d022f2c38d05)